core/mondoo-github-best-practices.mql.yaml

```diff
@@ -0,0 +1,188 @@
+        
+        ### Configure a GITHUB_TOKEN environment variable
+
+        You supply your personal access token to cnspec using the `GITHUB_TOKEN` environment variable. 
+
+        #### Linux / MacOS
+
+        ```bash
+        export GITHUB_TOKEN=<your personal access token>
+        ```
+
+        #### Windows 
+
+        ```powershell
+        $Env:GITHUB_TOKEN = "<personal-access-token>"
+        ```        
+
+        ## Scanning GitHub repositories
+        
+        To scan the configuration of a GitHub repository:  
+
+        ```bash
+        cnspec scan github repo <ORG_NAME/REPO_NAME>
+        ```
+
+        ## Join the community!
+
+        Our goal is to build policies that are simple to deploy, accurate, and actionable. 
+        
+        If you have any suggestions on how to improve this policy, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+    scoring_system: 2
+    specs:
+      - asset_filter:
+          query: |
+            asset.platform == "github-repo" 
+        scoring_queries:
+          mondoo-github-repository-best-practices-support-resources: null
+          mondoo-github-repository-best-practices-code-of-conduct: null
+          mondoo-github-repository-best-practices-include-authors: null
+          mondoo-github-repository-best-practices-readme-getting-started: null
+          mondoo-github-repository-best-practices-license: null
+queries:
+  - uid: mondoo-github-repository-best-practices-support-resources
+    title: Ensure repository has a support policy
+    severity: 30
+    docs:
+      desc: |
+        GitHub repositories should include a SUPPORT file to let people know how to get help with the project.
+
+        To direct people to specific support resources, you can add a SUPPORT file to your repository's root, docs, or .github folder. When someone creates an issue in your repository, they will see a link to your project's SUPPORT file.
+    refs:
+      - title: Adding support resources to your project
+        url: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project
+    query: |
+      // if a .github repo exists, then check if the SUPPORT.md is there
+      if ( github.organization.repositories.one(name == ".github") ) {
+        github.organization.repositories.where( name == ".github").all( 
+          files.one( name.downcase == "support.md")
+        ) || github.repository.files.one( name.downcase == "support.md")
+      } else {
+          github.repository.files.one( name.downcase == "support.md")
+      }
+  - uid: mondoo-github-repository-best-practices-code-of-conduct
+    title: Ensure repository has a code of conduct policy
+    severity: 30
+    docs:
+      desc: |
+        Open source code repositories should include a code of conduct. Including a code of conduct helps to clarify the project's values and principles.
```

Comment:
This should describe where we're expecting this file to be since we have a specific filename in the query